source/clear-linux/guides/maintenance/swupd-mixer-integration.rst

```diff
@@ -0,0 +1,155 @@
+.. _swupd-mixer-integration:
+
+Mixer tool enhancement
+######################
+
+Mixing means composing an operating system for a specific use case. The
+default build of |CLOSIA| provides options to install bundles for various
+server capabilities. If you want to supplement the default OS with
+functionality from other OS distributions, you can manually compose an OS
+using :ref:`the mixer tool <mixer>`.
+
+Software updater release 19900 integrates the mixer tool with the
+updater (:command:`swupd`) and provides a simpler way to mix your OS.
+
+Process overview
+****************
+
+To add a package that does not exist in the current |CL| bundle, perform the following:
+
+#. Create custom packages and bundles using the :file:`swupd-add-pkg` script.
+
+#. Add your desired content with the commands:
+
+   .. code-block:: bash
+
+      sudo swupd update
+      sudo swupd bundle-add <new_bundle>
+
+The upstream content is pulled from the normal URL and local content is added
+from a local update store.
+
+.. note::
+
+   The integrated mixer tool adds non-unique packages and replaces upstream
+   content with a local version. However, the updater exits if it finds
+   conflicts before creating the local content. You can override this behavior
+   by passing the :command:`--allow-mix-collisions` option.
+
```

Comment:
The "Mixer tool enhancement" and "Process overview" do not fit well to this document.  It's confusing.  What happened to @tmarcu original explanation of swupd-add-pkg?

Reply:
The original section called "how it works" was moved to later in this document since it is very detailed and may not be relevant for all users. Please see line 111 "swupd operational details" for that text.

Reply:
I agree w/ @bktan8 enhancement is not the right word here, it's just explaining a different tool/feature.